### ISSUE TITLE
Screen reader is not announcing the proper information with bot response.

### ISSUE THREAD

**Issue by 232amansingh** [microsoft/BotFramework-WebChat#5737]
### What is the PWD impact?

When a screen reader does not announce the proper information with a bot response, users relying on assistive technology cannot understand the content or context of the response. They may only hear generic output like “text” or “message” without the actual content, which prevents them from following the conversation.

### What browsers and screen readers do this issue affect?

Windows: Edge with Windows Narrator

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

Others or unrelated

### How to reproduce the issue?

Repro Steps: ​           
1. Open URL: Web Chat in edge browser.
2. Ask query 'Animationcard'.
3. Navigate to bot response in scan mode.
4. Observe the issue that Screen reader is not announcing the proper information with bot response. 

### What do you expect?

Screen reader should announce the proper information with bot response. 

### What actually happened?

Screen reader is not announcing the proper information with bot response. 

### WCAG reference

https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships

### Adaptive Card JSON

```json

```
